service/worker/migration/force_replication_workflow.go

```diff
@@ -502,7 +502,7 @@ func enqueueReplicationTasks(ctx workflow.Context, workflowExecutionsCh workflow
 // running simultaneously and the initial starting point.
 func NewQPSQueue(concurrentActivityCount int) QPSQueue {
 	return QPSQueue{
-		Data:    make([]QPSData, 0, concurrentActivityCount+1),
+		Data:    make([]QPSData, 0, max(0, 2*concurrentActivityCount+1)),
```

Comment:
how about making multiplier factor (number 2) here configurable so we can try out different number at least?